packages/preact-urql/src/hooks/useQuery.ts

```diff
@@ -175,7 +175,9 @@ export function useQuery<Data = any, Variables = object>(
 
   // This is the imperative execute function passed to the user
   const executeQuery = useCallback(
-    (opts?: Partial<OperationContext>) => update(makeQuery$(opts)),
+    (opts?: Partial<OperationContext>) => {
+      update(makeQuery$({ suspense: false, ...opts }));
```

Comment:
Nit: Shouldn't we move suspense overrides to the back so users can't accidentally invoke the feature.

```suggestion
      update(makeQuery$({ ...opts, suspense: false }));
```

Reply:
I believe it should still be possible to force enable it, since there may be legitimate uses to trigger Suspense manually in the render thread of lower components... somehow 🤷